bigquery_etl/cli/generate.py

```diff
@@ -0,0 +1,44 @@
+"""bigquery-etl CLI generate command."""
+import click
+from pathlib import Path
+import importlib.util
+from inspect import getmembers
+
+
+SQL_GENERATORS_DIR = "sql_generators"
+GENERATE_COMMAND = "generate"
+ROOT = Path(__file__).parent.parent.parent
+
+
+def generate_group():
+    """Create the CLI group for the generate command."""
+    commands = []
+    generator_path = ROOT / SQL_GENERATORS_DIR
+
+    for path in generator_path.iterdir():
+        if path.is_dir():
+            # get Python modules for generators
+            spec = importlib.util.spec_from_file_location(
+                path.name, (path / "__init__.py").absolute()
+            )
+            module = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(module)
```

Comment:
This is a bit icky, but I'm actually surprised at how little code is needed. I think it's a reasonable thing to do, as the contents of `sql_generators` are essentially plugins, and it does make sense to organize them outside the module itself.

But I think it would be worth having some more commentary right here to discuss that. I also think there should be a `sql_generators/README.md` to explain some context of what's going on in that dir and what the contract is.

Reply:
I added a few more comments and documentation to the README for this. Let me know if I should get more into details.